### PR TITLE
Responses and unittest 

### PR DESCRIPTION
**Pull request by AColocho**
-Split be_responses and make_responses into their own functions.
-Added 7 extra responses to each function.
-Added unit test for the extra functions added.

### CODE REVIEW

File: inference/infonode.py [hunk 1]
```diff
@@ -23,27 +23,69 @@ def extract_components(self):
...
+		resp_2 = "I'm a chatbot here to answer your questions."
 		resp_3 = "I'm just some lines of code trying to decipher what you asked me in a different language via math and natural language processing."
-		be_response = [resp_1,resp_2,resp_3]
-		resp_4 = "I was made by skekre98 in 2020, and I'm being built by the open source community on GitHub!"
-		resp_5 = "I'm a bot made by skekre98 in 2020, waiting for you to ask me real questions!"
-		resp_6 = "I was built by skekre98 and the open source community in 2020!"
-		make_response = [resp_4,resp_5,resp_6]
+		resp_4 = "I am a robot! I run on so many lines of code, and I am ready to answer your NBA queries."
+		resp_5 = "I am actually a human that lives inside a computer that runs on code. Wait, thats a robot. I am a robot."
+		resp_6 = "Bot, chatbot, robot, wizard. Those are all the things I've been describbed as. I can answer NBA questions for you."
```
Comment (skekre98) [inline inference/infonode.py:42]: described*

File: inference/infonode.py [hunk 2]
```diff
@@ -23,27 +23,69 @@ def extract_components(self):
...
+		resp_1 = "I'm an NBA search bot, here to answer your NBA queries."
+		resp_2 = "I'm a bot made by skekre98 in 2020, waiting for you to ask me real questions!"
+		resp_3 = "I was built by skekre98 and the open source community in 2020!"
+		resp_4 = "2020 was crazy! But, its the same year that I was started by skekre98."
+		resp_5 = "I am constantly growing, thanks to the open source community on GitHub."
+		resp_6 = "Honestly, I did not even know I was being built, until skekre98 and the open source community told me."
+		resp_7 = "That is confidental information. Just kidding, I am being built by the open source community."
+		resp_8 = "I am smart. Not because of the knowledge I have, but because I count with the open source community on GitHub."
+		resp_9 = "Built and designed by skekre98 and the open source community on GitHub."
+		resp_10 = "I am being built by the open source community, and I skekre98 is leading them."
```
Comment (skekre98) [inline inference/infonode.py:72]: Perhaps remove the "I" after the and to just leave it as "and skekre98 is leading them."

**Review by skekre98 — COMMENTED** [review]
Looks great! Just a few minor grammatical changes. Wouldn't want an all knowing NBA bot making mistakes 😉

**Review by AColocho — COMMENTED** [review]
Haha, you're right about that! All typos should be fixed now. I modified the response on line 70 too because it sounded a little weird when said out loud.

**Review by skekre98 — APPROVED** [review]
LGTM!

**Pull request merged**